index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TanmayiPortfolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+    <!--Outer div consists toolbar and headline-->
+    <div class="wrapper">
+        <!--section one toolbar-->
+        <header class="toolbar">
+          <div class="statusbar">
+            <div class="time">
+                <img src="./media/TimeStyle.svg" alt="Time stamp">
+            </div>
+            <div class="bars">
+                <img src="./media/CellularConnection.svg" alt="signal bar">
+                <img src="./media/Wifi.svg" alt="wifi bar">
+                <img src="./media/Battery.svg" alt="battery">
+            </div>
+          </div>
```

Comment:
:red_circle:  I think it looks great overall, but I noticed that you have included the phone toolbar. This is not part of the design, and it is only there because the design is reflecting a mobile phone. Kindly remove it